intro-HPC/ch_troubleshooting.tex

```diff
@@ -64,3 +64,27 @@ \section{Warning message when first connecting to new host}
 \else
 \firsttimeconnection
 \fi
+
+
+\section{Memory limits}
+
+To avoid jobs allocating too much memory, there are memory limits in place by default.
+It is possible to specify higher memory limits if your jobs require this.
+
+\subsection{How will I know if memory limits are the cause of my problem?}
+
+If your program fails with a memory-related issue, there is a good chance it failed
+because of the memory limits and you should increase the memory limits for your job.
+
+Examples of these error messages are: \verb|malloc failed|, \verb|Out of memory|,
+\verb|Could not allocate memory|
+or in Java: \verb|Could not reserve enough space for object heap|. Your program can
```

Comment:
for Java specifically, we should point to a separate section, see http://hpc.ugent.be/userwiki/index.php/User:Mem_limits#Java

Open an issue to add a best practice section on Java where this can be documented, and mention to refer to this section from here.

Same goes for MATLAB (see http://hpc.ugent.be/userwiki/index.php/Tips:Software:MATLAB)